### DISCUSSION TITLE
Disabling print on Console

### DISCUSSION THREAD

**Discussion by youareit125** [idaholab/moose#20752] [Q&A General] [answered]
Hi everyone,

I have been working on this code and have gotten to the point where everything works.
Thus, is there a way to disable printing ability on console? I have disabled the residual but still prints the timesteps, time, steady-state norm. I still would like to save the output at the end step.

Thank you!

```
[Debug]
show_var_residual_norms = false
[]

[Executioner]
type = Transient
automatic_scaling = true
end_time = 1.0e-3
solve_type = 'LINEAR'
dtmin = 1.0e-20
dtmax = 1.0e-6
steady_state_detection = true
steady_state_tolerance = 1e-30
nl_abs_tol = 1.0e-50
[TimeStepper]
type = IterationAdaptiveDT
growth_factor = 1.005
dt = 1.0e-12
[]
[]

[Preconditioning]
[smp]
type = SMP
full = true
[]
[]

[Outputs]
[hope_this_works]
type = Console
outlier_variable_norms = false
execute_scalars_on = none
clear_after_output = true
[]
[out]
type = CSV
execute_on = 'FINAL'
show = '{species}'
[]
[]
```

**Comment by GiudGiud** [comment, marked as answer]
Hello

You're on the right track with the [hope_this_works] console block. You should just execute it on FINAL or on TIMESTEP_END.

Note that you can disable a lot of printing with these 3 options in [Outputs]:
```
  print_linear_converged_reason = false
  print_linear_residuals = false
  print_nonlinear_converged_reason = false
```

Guillaume

**Reply by youareit125** [reply]
Thank you very much. I have them in my [hope_this_works] block. I have managed to remove some stuff on console. Is there a way to hide or not print time steps, time, dt or solve convergence?

**Reply by GiudGiud** [reply]
Not that I know of currently. There's planned changes in the future for setting a verbosity level, we'll make sure to have options to hit all these.

**Reply by youareit125** [reply]
Thank you very much!
